src/pages/RequisitionsPage.js

```diff
@@ -112,26 +126,51 @@ export class RequisitionsPage extends React.Component {
 
   renderCell(key, requisition) {
     switch (key) {
-      default:
-      case 'serialNumber':
-        return requisition.serialNumber;
       case 'entryDate':
         return requisition.entryDate.toDateString();
-      case 'numberOfItems':
-        return requisition.numberOfItems;
+      case 'otherStoreName':
+        return requisition.otherStoreName.name;
       case 'status':
         return formatStatus(requisition.status);
       case 'delete':
         return {
           type: 'checkable',
           icon: 'md-remove-circle',
-          isDisabled: requisition.isFinalised,
+          isDisabled: requisition.isFinalised || !requisition.isRequest,
         };
+      default:
+        return requisition[key];
     }
   }
 
+  renderRequisitionTypeToggle() {
+    return (
+      <View style={globalStyles.horizontalContainer}>
+        <PageButton
+          style={[localStyle.toggleButton, this.state.isRequestView ?
+                            localStyle.selectedButton : {}]}
+          text={buttonStrings.request_requisition}
+          onPress={this.toggleRequisitionViewType}
+        />
+        <PageButton
+          style={[localStyle.toggleButton, !this.state.isRequestView ?
+                            localStyle.selectedButton : {}]}
+          text={buttonStrings.response_requisition}
+          onPress={this.toggleRequisitionViewType}
+        />
+      </View>
+    );
+  }
```

Comment:
Will look into that tomorrow or refactor after the trip, I missed the ToggleBar component..